web_app/models/IC_model_ali2020.py

```diff
@@ -64,6 +76,51 @@ def eval_material_loss(self, time: float) -> float:
         Returns:
             float: The calculated material loss.
         """
-        material_loss = (0.00006 * self.parameters['C'] + 0.0008) * time + self.parameters['b']
+        material_loss = (0.00006 * self.parameters['C'] + 0.0008) * time*24*365 + self.parameters['b']
+        
         return material_loss
 
+    def _bilinear_interpolation(self, x, y, values, nacl_concentration, time):
+        """Perform bilinear interpolation."""
+        x1 = np.searchsorted(nacl_concentration, x) - 1
+        x2 = x1 + 1
+        y1 = np.searchsorted(time, y) - 1
+        y2 = y1 + 1
+        
+        x1 = np.clip(x1, 0, len(nacl_concentration) - 2)
+        x2 = np.clip(x2, 1, len(nacl_concentration) - 1)
+        y1 = np.clip(y1, 0, len(time) - 2)
+        y2 = np.clip(y2, 1, len(time) - 1)
+        
+        x1_val = nacl_concentration[x1]
+        x2_val = nacl_concentration[x2]
+        y1_val = time[y1]
+        y2_val = time[y2]
+        
+        Q11 = values[y1, x1]
+        Q21 = values[y1, x2]
+        Q12 = values[y2, x1]
+        Q22 = values[y2, x2]
+        
+        return (Q11 * (x2_val - x) * (y2_val - y) +
+                Q21 * (x - x1_val) * (y2_val - y) +
+                Q12 * (x2_val - x) * (y - y1_val) +
+                Q22 * (x - x1_val) * (y - y1_val)) / ((x2_val - x1_val) * (y2_val - y1_val))
```

Comment:
why?

Reply:
The experimental value is provided only for specific values, so interpolate and extrapolate to to extended time and other configurations.

Reply:
You can not do that. This would than bascially be a linear model in it self.

Reply:
True. Then the model can be compared with experimental results only till 1750 Hours. How do we proceed?